kinder/pkg/cri/host/archive.go

```diff
@@ -0,0 +1,196 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
```

Comment:
we should mention that this is forked from https://github.com/kubernetes-sigs/kind/blob/master/pkg/build/nodeimage/internal/container/docker/archive.go

and that the original package is internal

Reply:
added a doc.go file